### DISCUSSION TITLE
[SOLVED] Canales fantasma

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#28] [4. Bug Reports] [answered]
Buenas,

En producción hice desactivación de tareas usando el panel lateral con el botón DELETE y ahora me están volviendo a aparecer los canales pero al hacer click en ellos, no aparece nada en el canal. A veces sale el error que indica que el canal fue desactivado y otras sólo no aparece nada. Limpiar el cache no soluciona el problema. 

Hubo una actualización crítica en el momento de reportar esto (antes de las 14:00 de hoy) y luego creo que no se ha vuelto a repetir. De todos modos, dejo el precedente.

**Comment by rafaelwalter** [comment, marked as answer]
@SeIturra En el ultimo release subimos un fix que soluciona este problema en el frontend.

**Reply by guillevalin** [reply]
@SeIturra si vuelve a ocurrir coméntanos en qué versión ocurre, porque a veces por como funciona Cotalker, estás operando en una versión anterior a la última disponible (se actualiza en background con Service Workers).
